packages/@orchidui-vue/src/Charts/BarChart/OcBarChart.vue

```diff
@@ -1,10 +1,19 @@
 <template>
-  <div ref="barChart" class="w-full" />
+  <div class="w-full relative">
+    <div ref="barChart" class="h-full" />
+    <dummy-data v-if="dummyData" absolute>
+      <slot name="dummy-data-description">
+        Demo reports will be replaced once <br />
```

Comment:
@kutsaniuk having break is not a good idea. 
Ideally your styling should be able to handle any text length by using width and max lines. Anything longer than 3 lines will be truncated. So anyone using the component need not worry about line breaks

Reply:
@spacemonkey92 Good! Thanks